namui/src/namui/font/load_sans_typeface_of_all_languages.rs

```diff
@@ -91,22 +91,25 @@ mod tests {
         )
         .unwrap();
 
-        assert_eq!(
-            font_file_url_map,
-            HashMap::from([(
-                Language::Ko,
-                HashMap::from([
-                    (FontWeight::_100, "ko/NotoSansKR-Thin.otf".to_string()),
-                    (FontWeight::_300, "ko/NotoSansKR-Light.otf".to_string()),
-                    (FontWeight::_400, "ko/NotoSansKR-Regular.otf".to_string()),
-                    (FontWeight::_500, "ko/NotoSansKR-Medium.otf".to_string()),
-                    (FontWeight::_700, "ko/NotoSansKR-Bold.otf".to_string()),
-                    (FontWeight::_900, "ko/NotoSansKR-Black.otf".to_string()),
-                ])
-            )])
-        );
+        let answer = HashMap::from([(
+            Language::Ko,
+            HashMap::from([
+                (FontWeight::_100, "ko/NotoSansKR-Thin.otf".to_string()),
+                (FontWeight::_300, "ko/NotoSansKR-Light.otf".to_string()),
+                (FontWeight::_400, "ko/NotoSansKR-Regular.otf".to_string()),
+                (FontWeight::_500, "ko/NotoSansKR-Medium.otf".to_string()),
+                (FontWeight::_700, "ko/NotoSansKR-Bold.otf".to_string()),
+                (FontWeight::_900, "ko/NotoSansKR-Black.otf".to_string()),
+            ]),
+        )]);
+
+        assert_eq!(font_file_url_map, answer,);
```

Comment:
```suggestion
        assert_eq!(font_file_url_map, answer);
```
If trailing comma was not intended, why don't we change it like this?